project/plugins.sbt

```diff
@@ -12,7 +12,6 @@ addSbtPlugin("com.typesafe.sbt"    % "sbt-digest"         % "1.1.4")
 addSbtPlugin("org.scoverage"   % "sbt-scoverage"         % "2.0.9")
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
 addSbtPlugin("org.scalameta"   % "sbt-scalafmt"          % "2.5.2")
-addSbtPlugin("ch.epfl.scala"   % "sbt-bloop"             % "1.5.11")
```

Comment:
Removing bloop might be an issue until we isolate it properly to devs own workspaces.

Reply:
Yea I added it back in.